src/widgets/Cell.php

```diff
@@ -402,6 +402,7 @@ function(xhr) {
             'params' => ['status' => $newStatus],
             'options' => [
                 'class' => 'btn  btn-widget-control btn-' . (($widget->status && $published) ? 'success' : 'warning'),
+                'aria-label' => 'loading',
```

Comment:
"loading" is the wrong term here. Something like "Toggle visibility status" would be more appropriate here in my opinion. And why didn't you use Yii:t here and for any other aria-labels? Non-English speakers would benefit from this ;)